src/main.rs

```diff
@@ -48,40 +66,55 @@ fn main() {
         .setting(AppSettings::SubcommandRequired)
         .get_matches();
 
-    println!("Welcome to a round of 'battleship'");
+    ///////////////////////////////////////////////////////////////////////////////////////////////
+    ///                                         Start                                           ///
+    ///////////////////////////////////////////////////////////////////////////////////////////////
+    println!("{}", Yellow.paint("Welcome to a round of 'battleship'"));
 
     // default --> player vs. player
-    let mut mode = types::Mode::PvP;
+    let mut mode = Mode::PvP;
 
     match battleship.subcommand() {
         ("server", Some(server_args)) => {
             // required arguments
             let port = validate_port(server_args.value_of("port").unwrap());
-            let name = server_args.value_of("name").unwrap();
+            let name = String::from(server_args.value_of("name").unwrap());
 
             // optional arguments
             let mut size = BOARD_SIZE;
             if let Some(val) = server_args.value_of("size") {
+                println!(
+                    "{}",
+                    Red.paint("NOT IMPLEMENTED: Custom board size will be reset to 10x10")
+                );
                 match val.parse::<u8>() {
                     Ok(val) => size = val,
-                    Err(_) => size = helper::read_usize() as u8,
+                    Err(_) => size = util::read_usize() as u8,
                 }
```

Comment:
Yeah, but all printing methods and all board access is defined for 10x10 boards, so it is not really a feature yet ;)